tests/core/test_policies.py

```diff
@@ -38,10 +42,19 @@
     IntentMaxHistoryTrackerFeaturizer,
 )
 from rasa.shared.nlu.interpreter import RegexInterpreter
-from rasa.core.policies.policy import SupportedData, Policy, InvalidPolicyConfig
+from rasa.core.policies.policy import (
+    SupportedData,
+    Policy,
+    InvalidPolicyConfig,
+    PolicyGraphComponent,
+)
 from rasa.core.policies.rule_policy import RulePolicy
 from rasa.core.policies.ted_policy import TEDPolicy
-from rasa.core.policies.memoization import AugmentedMemoizationPolicy, MemoizationPolicy
+from rasa.core.policies.memoization import (
+    AugmentedMemoizationPolicyGraphComponent,
+    MemoizationPolicyGraphComponent,
+)
```

Comment:
how about doing this to avoid an unnecessary diff?
```suggestion
from rasa.core.policies.memoization import (
    AugmentedMemoizationPolicyGraphComponent as AugmentedMemoizationPolicy,
    MemoizationPolicyGraphComponent as MemoizationPolicy,
)
```